crml/cennzx-spot/src/lib.rs

```diff
@@ -51,8 +51,8 @@ pub trait Trait: system::Trait + generic_asset::Trait {
 	type Event: From<Event<Self>> + Into<<Self as system::Trait>::Event>;
 	/// A function type to get an exchange address given the asset ID pair.
 	type ExchangeAddressGenerator: ExchangeAddressFor<Self::AssetId, Self::AccountId>;
-	type BalanceToU128: From<<Self as generic_asset::Trait>::Balance> + Into<u128>;
-	type U128ToBalance: From<u128> + Into<<Self as generic_asset::Trait>::Balance>;
+	type BalanceToUnsignedInt: From<<Self as generic_asset::Trait>::Balance> + Into<LowPrecisionUnsigned>;
```

Comment:
It still bugs me that we need to have two types when they both bind to `Balance` ¯\_(ツ)_/¯

Reply:
Me too. I had a go at it. But it was at the time I had lots of compile errors and I needed to withdraw from some of my changes. I will have a second look at it to check if the solution is really simple. Otherwise, we may need to stand it until a later opportunity.

Reply:
As we discussed, this issues requires changes beyond this task and will be revisited later when we bring in our currency module and make cennzx depend on, instead of generic-assets.